base_ubl/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Base UBL",
-    "version": "13.0.2.0.1",
+    "version": "14.0.2.0.1",
```

Comment:
Refer guideline for module version:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

Reply:
fixed

Reply:
Please squash the related commit into the migration commit.

Reply:
@phuctranerp you resolved this comment but the separate commit for the version update is still present on the branch.

Reply:
@StefanRijnhart I am thinking if we should track the history of changes. Btw, I have squashed the commits. Thanks for your feedbacks